### PR TITLE
rspec post controller

### PR DESCRIPTION
**Pull request by justalak**
https://edu-redmine.sun-asterisk.vn/issues/22679

Screenshot from 2020-04-10 15-38-09
Screenshot from 2020-04-10 15-38-34

### CODE REVIEW

**Comment by justalak** [comment]
ready

File: spec/controllers/posts_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,143 @@
+require 'rails_helper'
```
Comment (manhbnt-0960) [inline spec/controllers/posts_controller_spec.rb:1]: "

File: spec/controllers/posts_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+RSpec.describe PostsController, type: :controller do
+  let!(:user){FactoryBot.create :user}
+  let!(:posts){FactoryBot.create_list :post, 10, user_id: user.id}
```
Comment (manhbnt-0960) [inline spec/controllers/posts_controller_spec.rb:5]: 10 đưa vào settings

**Comment by justalak** [comment]
ready

**Pull request merged**